app/src/atoms/buttons/OnDeviceDisplay/SmallButton.stories.tsx

```diff
@@ -15,44 +15,53 @@ const Template: Story<React.ComponentProps<typeof SmallButton>> = args => (
 
 export const Default = Template.bind({})
 Default.args = {
-  onClick: () => console.log('click'),
   buttonType: 'default',
   buttonText: 'Button text',
 }
 
 export const Alert = Template.bind({})
 Alert.args = {
-  onClick: () => console.log('click'),
   buttonType: 'alert',
   buttonText: 'Button text',
 }
 
 export const Alt = Template.bind({})
 Alt.args = {
-  onClick: () => console.log('click'),
   buttonType: 'alt',
   buttonText: 'Button text',
 }
 
-export const GhostLow = Template.bind({})
-GhostLow.args = {
-  onClick: () => console.log('click'),
-  buttonType: 'ghostLow',
+export const TertiaryLowLight = Template.bind({})
+TertiaryLowLight.args = {
+  buttonType: 'TertiaryLowLight',
```

Comment:
Storybook shows error because the first letter of the buttonType.
```suggestion
  buttonType: 'tertiaryLowLight',
```

Reply:
Oops omg i could not figure out why it wasn't working. thank you!